optaplanner-core/src/main/java/org/optaplanner/core/api/score/stream/package-info.java

```diff
@@ -16,6 +16,50 @@
 
 /**
  * The {@link org.optaplanner.core.api.score.stream.ConstraintStream} API:
- * a way to define constraints for {@link org.optaplanner.core.api.score.Score} calculation.
+ * a way to define constraints for {@link org.optaplanner.core.api.score.Score} calculation. Throughout this
+ * documentation, we will be using the following terminology:
```

Comment:
This is pointless. Nobody reads the package javadocs :(

Defining this terminology can happen in the user documentation, as they 'll read that diagonally first, before coding and seeing javadocs. Let's remove it here and move it to there, as it's a maintenance burden here.

Reply:
Update: Let's move it to ConstrainStream's javadoc.